setup/install_gatling/20_foss_install/20_install_puppet.rb

```diff
@@ -0,0 +1,24 @@
+test_name 'install Puppet for a scale environment' do
+  skip_test 'Installing PE, not FOSS' unless ENV['BEAKER_INSTALL_TYPE'].casecmp('foss') == 0
+
+  # step "Upgrade nss to version that is hopefully compatible with jdk version puppetserver will use." do
+    # nss_package=nil
+    # variant, _, _, _ = master['platform'].to_array
+    # case variant
+    # when /^(debian|ubuntu)$/
+    #   nss_package_name="libnss3"
+    # when /^(redhat|el|centos)$/
+    #   nss_package_name="nss"
+    # end
+    # if nss_package_name
+    #   master.upgrade_package(nss_package_name)
+    # else
+    #   Beaker::Log.warn("Don't know what nss package to use for #{variant} so not installing one")
+    # end
+  # end
```

Comment:
do we need this?

Reply:
nope, I just realized there was some unfinished stuff in this commit.  I was going to re-name these tests and remove the stuff we no longer needed and didn't do that. I'll update this.